src/components/Navbar/Navbar.styled.ts

```diff
@@ -37,20 +38,21 @@ export const NavLinks = styled.div`
 
 export const Button = styled.button`
 	background-color: ${themes.light.background};
+	font-family: ${typography.fontFamily.secondary};
```

Comment:
Zamiast tworzyć customowy obiekt typography mógłbyś po prostu rozszerzyć obiekt themes 

https://dev.to/aromanarguello/how-to-use-themes-in-styled-components-49h

Tutaj tutorial jak robić theme w styled components